components/kyma-environment-broker/internal/metrics/operations_collector.go

```diff
@@ -15,68 +17,114 @@ const (
 	subsystem = "operations"
 )
 
-// OperationsStatsGetter provides metrics, which shows how many operations were done:
-// - compass_keb_operations_provisioning_failed_total
-// - compass_keb_operations_provisioning_in_progress_total
-// - compass_keb_operations_provisioning_succeeded_total
-// - compass_keb_operations_deprovisioning_failed_total
-// - compass_keb_operations_deprovisioning_in_progress_total
-// - compass_keb_operations_deprovisioning_secceeded_total
+// OperationsStatsGetter provides metrics, which shows how many operations were done for the following plans:
+
+// AZURE:
+
+// - compass_keb_operations_azure_provisioning_failed_total
+// - compass_keb_operations_azure_provisioning_in_progress_total
+// - compass_keb_operations_azure_provisioning_succeeded_total
+// - compass_keb_operations_azure_deprovisioning_failed_total
+// - compass_keb_operations_azure_deprovisioning_in_progress_total
+// - compass_keb_operations_azure_deprovisioning_succeeded_total
+
+// AZURE TRIAL:
+
+// - compass_keb_operations_azure_trial_provisioning_failed_total
+// - compass_keb_operations_azure_trial_provisioning_in_progress_total
+// - compass_keb_operations_azure_trial_provisioning_succeeded_total
+// - compass_keb_operations_azure_trial_deprovisioning_failed_total
+// - compass_keb_operations_azure_trial_deprovisioning_in_progress_total
+// - compass_keb_operations_azure_trial_deprovisioning_succeeded_total
+
+// AZURE LITE:
+
+// - compass_keb_operations_azure_lite_provisioning_failed_total
+// - compass_keb_operations_azure_lite_provisioning_in_progress_total
+// - compass_keb_operations_azure_lite_provisioning_succeeded_total
+// - compass_keb_operations_azure_lite_deprovisioning_failed_total
+// - compass_keb_operations_azure_lite_deprovisioning_in_progress_total
+// - compass_keb_operations_azure_lite_deprovisioning_succeeded_total
 type OperationsStatsGetter interface {
 	GetOperationStats() (internal.OperationStats, error)
```

Comment:
just a thought, we can make a struct OperationStatsByPlan which would be `map[string][string]struct{}` type of data.